File: Components/Document Server/VB/DocsMainDemo/Modules/About.vb
Imports Microsoft.VisualBasic
Imports System.Drawing
Imports DevExpress.Tutorials
Imports DevExpress.Utils.About

Namespace DevExpress.Docs.Demos
	Partial Public Class About
		Inherits ucOverviewPage
		Public Sub New()
			InitializeComponent()
		End Sub

		Protected Overrides ReadOnly Property Line1Text() As String
			Get
				Return ".NET Document Server"
			End Get
		End Property
		Protected Overrides ReadOnly Property Line2Text() As String
			Get
				Return "Read and Write XLSx, XLS, DOCx, DOC and PDF files"
			End Get
		End Property
		Protected Overrides ReadOnly Property Line3Text() As String
			Get
				Return "Create Snap Reports, Generate Barcodes and Compressed Archives"
			End Get
		End Property
		Protected Overrides ReadOnly Property Line4Text() As String
			Get
				Return votedVSM
			End Get
		End Property
	End Class
End Namespace

File: Components/Document Server/CS/DocsMainDemo/Modules/About.cs
using System.Drawing;
using DevExpress.Tutorials;
using DevExpress.Utils.About;

namespace DevExpress.Docs.Demos {
    public partial class About : ucOverviewPage {
		public About() {
			InitializeComponent();
		}

        protected override string Line1Text { get { return ".NET Document Server"; } }
        protected override string Line2Text { get { return "Read and Write XLSx, XLS, DOCx, DOC and PDF files"; } }
        protected override string Line3Text { get { return "Create Snap Reports, Generate Barcodes and Compressed Archives"; } }
        protected override string Line4Text { get { return votedVSM; } }
	}
}

